mesa-quilt

```diff
@@ -77,7 +77,8 @@ if [ -d "$d/nx-X11/extras" ]; then
 				if [ -n "$patchfile" ] && [ -r "../Mesa.patches/$patchfile" ]; then
 					echo "Removing patch ../Mesa.patches/$patchfile"
 					patch -R -p1 < "../Mesa.patches/$patchfile"
-					sed -i "/$patchfile/d" "../Mesa.patches/.applied-patches"
+					sed -e "/$patchfile/d" "../Mesa.patches/.applied-patches" > "../Mesa.patches/.applied-patches.tmp"
```

Comment:
Okay, much better.

There's still a big caveat, though: sed interprets `${patchfile}` as a regexp and there's no way to turn that off. We should be mostly fine if the filenames are sane, but once they contain chars like `[`, `]` and others, this will fail badly. Even without "evil" characters, each full stop character in filenames will be interpreted as "any character", which might lead to problems.

We could work around that, but it's too complicated for that PR. Will clean that up some time later.